tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -128,6 +128,11 @@ if [[ $BUILD_EXTR_PAR = *--enable_training_python_frontend_e2e_tests* ]]; then
     # DOCKER_RUN_PARAMETER="$DOCKER_RUN_PARAMETER -u0"
 fi
 
+if [[ $BUILD_EXTR_PAR = *--enable_training_pipeline_e2e_tests* ]]; then
+    DOCKER_RUN_PARAMETER="$DOCKER_RUN_PARAMETER --volume /bert_ort:/bert_ort \
```

Comment:
is it possible to avoid hardcoding the host path?

Reply:
I think it is fine using hard code path here: the path is a well established place for model accessing and is being used in other tests for model accessing too. For example: orttraining-linux-gpu-perf-test-ci-pipeline.yml